crates/astria-sequencer-relayer/src/relayer/write/conversion.rs

```diff
@@ -92,3 +120,22 @@ pub(super) fn convert(block: SequencerBlock) -> eyre::Result<Converted> {
         },
     })
 }
+
+fn brotli_compressed_bytes(data: &[u8]) -> eyre::Result<Vec<u8>> {
```

Comment:
Nit: as for the method used inside conductor using a verb is more idiomatic:
```suggestion
fn brotli_compress_bytes(data: &[u8]) -> eyre::Result<Vec<u8>> {
```